src/sentry/tagstore/base.py

```diff
@@ -396,16 +396,14 @@ def update_group_tag_key_values_seen(self, project_id, group_ids):
         """
         raise NotImplementedError
 
-    def get_group_tag_keys_and_top_values(self, project_id, group_id, environment_id, user=None):
-        from sentry.api.serializers import serialize
+    def get_group_tag_keys_and_top_values(self, project_id, group_id, environment_id, keys=None, limit=10):
+        # TODO apply limit to keys? or just values per key
```

Comment:
It is a little confusing that passing `limit=5` could return 500 keys (with 5 values per key). If we chose to only limit the values maybe we should name it `value_limit` or something.